src/ResourceManager/Network/Commands.Network/NetworkWatcher/ConnectionMonitor/SetAzureNetworkWatcherConnectionMonitorCommand.cs

```diff
@@ -0,0 +1,262 @@
+            ParameterSetName = "SetByName")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Location of the network watcher.",
+            ParameterSetName = "SetByLocation")]
+        [LocationCompleterAttribute]
+        [ValidateNotNull]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Resource ID.",
+            ParameterSetName = "SetByResourceId")]
+        [ValidateNotNull]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Connection monitor object.",
+            ParameterSetName = "SetByInputObject")]
+        [ValidateNotNull]
+        public PSConnectionMonitorResult InputObject { get; set; }
+
+        [Alias("ConnectionMonitorName")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The connection monitor name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The ID of the connection monitor source.")]
+        [ValidateNotNullOrEmpty]
+        public string SourceResourceId { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Monitoring interval in seconds.")]
+        [ValidateNotNullOrEmpty]
+        public int? MonitoringIntervalInSeconds { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Source port.")]
+        [ValidateNotNull]
+        [ValidateRange(1, int.MaxValue)]
+        public int SourcePort { get; set; }
+
+        [Parameter(
+             Mandatory = false,
+             HelpMessage = "The ID of the connection monitor destination.")]
+        [ValidateNotNullOrEmpty]
+        public string DestinationResourceId { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The Ip address of the connection monitor destination.")]
+        [ValidateNotNullOrEmpty]
+        public string DestinationAddress { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Destination port.")]
+        [ValidateNotNull]
+        [ValidateRange(1, int.MaxValue)]
+        public int DestinationPort { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Auto start.")]
+        [ValidateNotNullOrEmpty]
+        public bool? AutoStart { get; set; }
```

Comment:
@irrogozh same comment about changing this to `Stopped` (or anything else you can come up with)

Reply:
Done